pytorch_pfn_extras/training/extensions/lr_scheduler.py

```diff
@@ -48,6 +48,9 @@ class LRScheduler(extension.Extension):
         stepper (callable): Function that performs the step on
             the scheduler.
         trigger: Frequency to call this extension.
+        wait_for_first_optimizer_step (bool): Wait until optimizer.step is called
+            before invoking scheduler.step. This can address the issue where
+            optimizer.step is not called from the first iteration when using GradScaler.
```

Comment:
```suggestion
        wait_for_first_optimizer_step (bool): Wait until `optimizer.step()` is called
            before invoking `scheduler.step()`. This can address the issue where
            `optimizer.step()` is not called from the first iteration when using GradScaler.
```